code/modules/cargo/bounties/reagent.dm

```diff
@@ -175,7 +175,6 @@
 		/datum/reagent/consumable/frostoil,\
 		/datum/reagent/toxin/slimejelly,\
 		/datum/reagent/teslium/energized_jelly,\
-		/datum/reagent/toxin/skewium,\
```

Comment:
can you just comment out that line instead of deleting it?
line should be
```//	/datum/reagent/toxin/skewium,\ austation -- removes skewium from possible bounties```

Reply:
fixed